pyproject.toml

```diff
@@ -2,5 +2,5 @@
 requires = [
     "setuptools>=58.0.0",
     "wheel",
-    "Cython"
+    "Cython==3.1.0"
```

Comment:
Why the pinning?

Reply:
Sorry, that was for testing! Will revert